integration_tests/waypoints.cpp

```diff
@@ -64,276 +37,171 @@ TEST_F(SitlTest, MissionAddWaypointsAndFly)
     Device &device = dc.device();
 
     while (!device.telemetry().health_all_ok()) {
-        std::cout << "waiting for device to be ready" << std::endl;
+        LogInfo() << "waiting for device to be ready";
         std::this_thread::sleep_for(std::chrono::seconds(1));
     }
 
-    std::cout << "Device ready, let's start" << std::endl;
+    LogInfo() << "Device ready, let's start";
 
```

Comment:
Perhaps here add line "Creating and uploading mission"

Reply:
Sure.